.ipynb_checkpoints/Gold-Dataset-EDA-checkpoint.ipynb

```diff
@@ -0,0 +1,1584 @@
+{
```

Comment:
<p>We can see that each year is missing about 100 days</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/shamelj/Precious-Metals-Forecaster/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>